content/modules/reports/pages/quick-start.adoc

```diff
@@ -0,0 +1,315 @@
+[[quick_start]]
+= Quick Start
+
+This section contains instructions about using reports wizard to create and run reports. For an easy start, the reports add-on comes with the reports wizard, which is a visual tool for quick report creation, including both xref:creation/data-structure.adoc[data structure] and xref:creation/templates.adoc[template design].
+
+[[project_setup_report]]
+== Project Setup
+
+Installing add-ons::
+
+First, create a new Jmix application and then follow the xref:index.adoc#installation[instruction] to install the reports add-on and charts add-on.
+
+Creating entities and screens::
+
+As an example, we will create a simple application `Library` with several simple entities.
```

Comment:
What about uploading the project on GitHub? We have an organization for sample projects: https://github.com/jmix-projects